specs/PushNotifications/PushNotifications-spec.md

```diff
@@ -84,38 +84,48 @@ Power Management compliance.
   * It will subscribe to a In-memory Push event handler hanging off the retrieved Push Channel component.
     * The app should ideally handle the event by setting the Handled property to true to prevent Background Activation from <br/>
     launching a new process.
-
+* If the application is unpackaged, it will use the ProtocolActivator to register the activator instead. The app will also need to 
+call into the bootstrapper API to have access to the Windows App SDK. Click [here](https://docs.microsoft.com/en-us/windows/apps/windows-app-sdk/tutorial-unpackaged-deployment?tabs=cpp) for information on how to build and 
```

Comment:
Please word-wrap. Use VSCode's prettier extension which will pull in the .editorconfig for consistency.